src/collective/cover/widgets/textlinessortable.py

```diff
@@ -80,7 +80,7 @@ def get_custom_title(self, uuid):
             return title
         # If didn't find, get object title
         obj = uuidToObject(uuid)
-        return obj.Title()
+        return obj.Title().decode('utf-8')
```

Comment:
is using `safe_unicode()` not, well, safer?  I think (but not sure) there's a danger that obj.Title() may be e.g. u'\xc6\xb5' (as opposed to '\xc6\xb5') in which case using `decode('utf-8')` will break.

Reply:
Yes, thanks, I'll change that.   I had a nagging suspicion.